compiler/optimizer/VPHandlers.cpp

```diff
@@ -9665,7 +9717,7 @@ static TR::Node *constrainIfcmpeqne(OMR::ValuePropagation *vp, TR::Node *node, b
          }
       }
 
-   if (virtualGuardConstraint && virtualGuardWillBeEliminated)
```

Comment:
Since this makes `virtualGuardConstraint` unused, would you mind removing its declaration as well?

Reply:
Done